### DISCUSSION TITLE
Discus the first Cormas Road Map

### DISCUSSION THREAD

**Discussion by ElCep** [cormas/cormas#496] [Announcements]
## 📢 Discus the first Cormas Road Map (CRM)
Cormas has evolved a lot in the last few months. And a first stable version could be released soon.

To move in this direction, we should be able to quickly validate a roadmap together. 

During the meeting of September 24th, we proposed to discuss all week here the minimal features we expect from this first "stable" version of cormas. 
The objective would be to make a list of features to implement and bugs to solve to be able to propose Cormas in production. 

The meeting on October 1st will be the occasion to validate a first version of proposals.

Feel free to install the latest version, list the bugs (in the Github issues, but also here in the discussion) and propose essential features for this step.

 Remember that this is a community we build together 💪.

**Comment by pierbo** [comment]
Yes, good initiative, thank you Etienne!

**Comment by pierbo** [comment]
I would say that, for version 1.0, simples simulations like ECEC (without interaction) should work. This means:
- load and save a model (no version of the model for the moment),
- Graphics of the global probes, with update (of the axes) at each time step,
- ...
The UML Class drag editor is almost ready. It wasn't on priority 1, but now that it is quasi done, it would be a good advantage to have it for this release, no?
